### DISCUSSION TITLE
there are wrong links in the async dependencies section

### DISCUSSION THREAD

**Discussion by yuminee** [fastapi/fastapi#9254] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# No code
```

### Description

In the FastAPI documentation, there are wrong links in the [async dependencies section](https://fastapi.tiangolo.com/async/#dependencies). 
The link provided to sub-dependencies and dependencies are incorrect and should be updated to https://fastapi.tiangolo.com/tutorial/dependencies/sub-dependencies and https://fastapi.tiangolo.com/tutorial/dependencies

### Operating System

macOS

### FastAPI Version

0.88.0

### Python Version

Python 3.8.15

**Comment by hasansezertasan** [comment, marked as answer]
I works fine.
